packages/frontend-shared/src/components/Tooltip.cy.tsx

```diff
@@ -1,24 +1,58 @@
 import Tooltip from './Tooltip.vue'
 
-const slotContents = {
-  popper: () => <span>tooltip</span>,
-  default: () => <span class="default-slot">hello</span>,
-}
+const slotContents = (x: number) => ({
+  popper: () => <span>Tooltip {x}</span>,
+  default: () => <span class={`default-slot${x}`}>hello</span>,
+})
 
 describe('<Tooltip />', () => {
   it('playground', () => {
     cy.mount(() => {
       return (
         <div class="p-4 w-100px">
           {/* @ts-ignore */}
-          <Tooltip v-slots={slotContents} placement="right" />
+          <Tooltip v-slots={slotContents(0)} placement="right" />
         </div>
       )
     })
 
-    cy.get('.default-slot')
+    cy.get('.default-slot0')
+    .realHover()
+
+    cy.contains('Tooltip 0')
+  })
+
+  it('playground - interactive', { viewportHeight: 400 }, () => {
```

Comment:
Since interactive is intended to have a different appearance than regular tooltips, could we add a percy snapshot to this and the other test, and update the test descriptions?